knowledge-base/migrate-to-the-html5-viewer-from-the-old-asp.net-webforms-reportviewer-control.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Migrate to the HTML5 Viewer from the old ASP.NET WebForms ReportViewer control
```

Comment:
Can we word this in a more simpler way? For example: "Upgrade your ASP.NET WebForms ReportViewer control to the new HTML5 Viewer"